src/public/services/BillingService.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios'
+
+import { LoginStatistic } from 'src/types/login'
+
+// Exported for testing
+export const BILLING_ENDPOINT = '/billing'
+type BillingResult = LoginStatistic
```

Comment:
This looks dodgy - BillingResult should be typed as a _data transfer object_ or DTO, which describes the shape of data returned by the backend. You can refer to _AnalyticStatsDto_ type to see how it is being used in both frontend and backend - this is to ensure both client and server have a common contractual understanding of the shape of data.

Reply:
fixed - adding dto for `exampleService` and `billingService` fe/be. i think we do need a utility type that is just `PossibleError<T>` that wraps `T` to be either `T | ErrorDto` just for convenience. 

also, i wrote the explicit parameter documentation on the dto rather than the function itself. function just has a high level view of the actual parameters.